.travis.yml

```diff
@@ -4,6 +4,7 @@ python:
   - "3.5"
   - "3.6"
   - "3.7"
+  - "3.8"
```

Comment:
The `setup.py` lists supported versions as 3.6 to 3.10 (which is good!) so the CI config should match.